test/files/jvm/scala-concurrent-tck.scala

```diff
@@ -90,6 +90,25 @@ trait FutureCallbacks extends TestBase {
     promise.success(-1)
   }
 
+  def stressTestNumberofCallbacks(): Unit = once {
```

Comment:
I'll pull the code out to a separate project and manually move the code. Seems like the least intrusive way.

OTOH it would probably not be a completely bad idea to have a metaproject with benches that you can run on designated versions + specific commithash.